enterprise/admin/views.py

```diff
@@ -229,28 +232,47 @@ def _enroll_users(cls, enterprise_customer, emails, course_id, mode, request):
             course_id: The ID of the course in which we want to enroll
             mode: The enrollment mode the users will be enrolled in the course with
             request: The HTTP request the enrollment is being created by
+            notify: Whether to notify (by email) the users that have been enrolled
         """
         enrolled = []
         non_existing = []
         failed = []
         enrollment_client = EnrollmentApiClient()
+        course_details = CourseApiClient().get_course_details(course_id)
+        reverse_method = get_reverse_by_site(request, enterprise_customer.site)
+        course_url = reverse_method('about_course', args=(course_id,))
```

Comment:
> In re: your code suggestion. I'd rather not do it that way; it assumes that the site is accessible over https.

Yeah, but any production site should always be using HTTPS (and e.g. all OpenCraft and edX sandboxes do, and certainly anyone selling "enterprise" offerings would be using HTTPS). The only time HTTPs is not used AFAIK would be on devstacks, and then no emails are getting sent anyways - just logs to the console, so I assume it's not a big deal if the URLs shown in the console logged emails use the wrong protocol. But if that offends your sensibilities, it's fine to leave as is :)